inappmessaging/USERGUIDE.md

```diff
@@ -232,6 +232,21 @@ Custom event class:
 InAppMessaging.instance().logEvent(CustomEvent("search").addAttribute("keyword", "book").addAttribute("number_of_keyword", 1))
 ```
 
+### #9 Campaign's context (advanced feature)
```

Comment:
Please move this to another section (i.e. Advance Feature or Advance Usage) and add a simple description for the "context". Like "Context are used to determine if the campaign should be displayed...".